docker/apachespark/long_running.py

```diff
@@ -76,7 +76,15 @@ def run() -> List[Tuple[int, int]]:
     print("Job results are:", result.get())
     sc.stop()
 
+def signal_handler(signal, frame):
+    global interrupted
+    interrupted = True
 
 if __name__ == "__main__":
-    for i in range(100):
+    signal.signal(signal.SIGINT, signal_handler)
+    global interrupted
+    interrupted = False
+    while True:
         main()
```

Comment:
Does main does some backoff? Anyway, LGTM.

Reply:
as I understand it, this script submits 100 times a job to spark with a random sleep in them. This change runs the test constantly until the container receives a kill signal.